#name exception
a=10
c=a+b
print("sum=",c)